config/initializers/15_string_output_safety.rb

```diff
@@ -1,6 +1,6 @@
 
 
-if Rails.version < '5'
+if Rails.version < '6.1'
```

Comment:
Corrected the Rails version we're backporting from

Reply:
They're needed, they're backports for CVEs that are corrected in later versions of Rails. When I went through this PR, I evaluated whether they're needed and tried to update the version guards. So, in this case, the CVE was fixed in a version of Rails 6.1 so we need it until then.